inventory/inventory.go

```diff
@@ -0,0 +1,72 @@
+package inventory
+
+import (
+	"github.com/opencontrol/compliance-masonry/models"
+	"github.com/opencontrol/compliance-masonry/tools/certifications"
+)
+
+// Inventory maintains the inventory of all the controls within a given workspace.
+type Inventory struct {
+	*models.OpenControl
+	masterControlList       map[string]models.Control
+	actualSatisfiedControls map[string]models.Satisfies
+	MissingControlList      map[string]models.Control
+}
+
+// standardAndControlString makes a string from the standard and the control.
+// This is helpful for functions that want to create unique keys consistently.
+func standardAndControlString(standard string, control string) string {
```

Comment:
change of plans. Going with option 3. when creating the standard-control key with `models.Satisfies` for the actual documented controls, I just need that one object. With the master list, not only do i need `models.Control`, but i also need the enclosing struct. And that enclosing struct contains many Controls. So it gets a little complicated.